packages/vuetify/src/composables/outlined.ts

```diff
@@ -0,0 +1,31 @@
+// Utilities
+import { computed } from 'vue'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+export interface OutlinedProps {
+  outlined?: boolean | string
```

Comment:
Specify exact values, not `string`

Reply:
there are quite a few, border is bananas. not sure how I would do this. for now going to resolve this feedback.